gh-actions/common/build-debian/action.yml

```diff
@@ -46,69 +55,194 @@ runs:
       run: |
         set -eu
 
-        echo "::group::Install devscripts"
-        DEBIAN_FRONTEND=noninteractive sudo apt update
-        DEBIAN_FRONTEND=noninteractive sudo apt install -y devscripts
-        echo "::endgroup::"
-
-        echo "::group::Append commit SHA to local version"
+        echo "::group::Create local version with commit and docker container"
         cd '${{ inputs.source-dir }}'
-        sanitized_docker=$( echo "${{ inputs.docker-image }}" | sed 's/://' )
-        debchange --local "+${sanitized_docker}+${{ github.sha }}" "Github build. Job id: ${{ github.run_id }}. Attempt: ${{ github.run_number }}."
+
+        # Short commit to avoid "package-has-long-file-name"
+        echo VERSION_REF=$(echo ${{ github.sha }} | cut -c1-8)  >> $GITHUB_ENV
+
+        echo DEBFULLNAME="GitHub actions runner" >> $GITHUB_ENV
+        echo DEBEMAIL="noreply@github.com" >> $GITHUB_ENV
+
+        if git status --porcelain &>/dev/null; then
+          echo VERSION_REF=$(git rev-list --count HEAD)+$(git rev-parse --short HEAD) >> $GITHUB_ENV
+          echo DEBFULLNAME="$(git log -1 --format='%an' HEAD) - GH Action" >> $GITHUB_ENV
+          echo DEBEMAIL="$(git log -1 --format='%ae' HEAD)" >> $GITHUB_ENV
+        fi
         echo "::endgroup::"
 
+    - name: Prepare source package
+      uses: kohlerdominik/docker-run-action@v1.1.0
```

Comment:
So, not really very updated but still more recent than the more popular https://github.com/addnab/docker-run-action that also does not provide many inputs, and so it requires us to use more `options` which is still fine but more prone to error and less clean IMHO.

So, I had reviewed the code and it's quite simple and fine from my POV.

In case we can also fork it to something even simpler in bash that does the same, but it seems an overkill to me.